global CSS

GuilhermeAugustoP opened the pull request:
first task

GuilhermeAugustoP committed:
global CSS

first task

TyAssis commented on public/index.html:
```diff
@@ -24,7 +24,10 @@
-    <title>React App</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
```
dúvida: as fontes Roboto e Mea Culpa já não vem nos navegadores por padrão? 🤔

GuilhermeAugustoP replied:
Não sei... vou pesquisar sobre. Entretanto esse index.html é o arquivo padrão q vem quando fazemos o CRA.  Essas linhas de código eu copiei do projeto da lista de contatos. Lembra que em uma das mentorias vc me falou que a fonte que estava aparecendo não era a Roboto, ai vc me mostrou o fonts google API, e me ensinou como utilizar uma fonte em um projeto sem precisar de instalar?

TyAssis commented on src/App.js:
```diff
@@ -1,25 +1,28 @@
-          Learn React
-        </a>
-      </header>
+    <div className='Body'>
```
Não acho que a gente vá precisar de uma caixona maior pra agrupar tudo.
Pense que a página do Navegador já é onde a gente coloca as coisas.
Adicionar uma div envolta do projeto todo vai dificultar mais do que ajudar

TyAssis replied:
Isso não tem a ver com a task exatamente, então se for só a fim de teste, beleza ficar assim por enquanto👍🏼

GuilhermeAugustoP replied:
> Não acho que a gente vá precisar de uma caixona maior pra agrupar tudo. Pense que a página do Navegador já é onde a gente coloca as coisas. Adicionar uma div envolta do projeto todo vai dificultar mais do que ajudar

Entendi, mas nesse caso o que você sugere que eu faça? Porque a função so aceita retornar uma div, certo? Tinha sido por isso que fiz uma div maior, para englobar as outras.

TyAssis commented on src/index.css:
```diff
@@ -1,11 +1,101 @@
+  --font-size: 20px;
+  --primary-font-color: #4c4c4c;
+  --secondary-font-color: #ffffff;
+}
```
✨

TyAssis commented on src/index.css:
```diff
@@ -1,11 +1,101 @@
+  background: #ffffff;
+
+}
+
```
Como esse aqui é o arquivo do CSS Global Style, recomendo mover esses outros estilos para outro lugar.
Ou excluir se não forem mais usados.

TyAssis reviewed (commented):
Quando tiver mais alterações, me chama de novo no símbolo de reciclagem que fica do lado da minha carinha ali na PR

GuilhermeAugustoP committed:
working on Global CSS

-Added Mea Culpa font family
- Global CSS splited into a new file
- tried to create a p class and a secondary one. Although the secondary class was overlapped by p.
-Style from external div removed
